types/testing-library__jest-dom/test/testing-library__jest-dom-global-tests.ts

```diff
@@ -0,0 +1,67 @@
+import { ValidElement } from "@testing-library/jest-dom";
```

Comment:
This is not global test. 

From https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-i-write-definitions-for-packages-that-can-be-used-globally-and-as-a-module
>  The global test file should exercise the definition according to how it would be used in a script loaded on a web page where the library is available on the global scope - in this scenario you should not specify an import statement.

Reply:
Fixed, thank you.